src/common/imageio_libraw.h

```diff
@@ -0,0 +1,50 @@
+/*
+    This file is part of darktable,
+    Copyright (C) 2021 darktable developers.
+
+    darktable is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    darktable is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with darktable.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef DT_IMAGEIO_LIBRAW_H
+#define DT_IMAGEIO_LIBRAW_H
+
+#include "common/image.h"
+#include "common/mipmap_cache.h"
+
+#ifdef HAVE_LIBRAW
+int dt_libraw_lookup_makermodel(const char *maker, const char *model,
```

Comment:
gboolean instead of int to match rawspeed?

Reply:
Ops, forgot this one. Fixed now.